spec/lib/active_mocker/generate_spec.rb

```diff
@@ -95,11 +97,31 @@
       end
     end
 
+    describe "#active_record_models" do
+      let(:models_dir) { File.join(File.expand_path("../../", __FILE__), "models") }
```

Comment:
Can you do the same thing you did on line 12 with only using `File.expand_path`?